### ISSUE TITLE
Cannot add items to list converted from paragraph

### ISSUE THREAD

**Issue by tiotdev** [codex-team/editor.js#995]
To reproduce:
1. Write a paragraph
2. Convert to list
3. Press the return key to add a new item to the list - it does not work

**Comment by next-direction** [comment]
Just played around with this. First thing I recognized is that it doesn‘t matter what kind of conversion is done. Paragraph to heading also leads to non-working lists.

I think there is an event handler inside the flipper component which stops propagation of the keys that are registered in the component.

**Comment by next-direction** [comment]
After further investigating this issue, turns out there is a logic in `src/components/modules/toolbar/inline.ts` where the order should be the other way around:

```
this.Editor.Listeners.on(this.nodes.conversionToggler, 'click', () => {
  this.Editor.ConversionToolbar.toggle((conversionToolbarOpened) => {
    if (conversionToolbarOpened) {
      this.flipper.deactivate();
    } else {
      this.flipper.activate();
    }
  });
});
```

When convertion toolbar opens, it should activate, when it is closed, it should deactivate. Currently it is the exact opposite.

**Comment by neSpecc** [comment]
Resolved by #1001 
Released in 2.17.0

Thanks to @aablinov and @khaydarov
